install/install.sh

```diff
@@ -224,7 +236,7 @@ detect_target() {
     target="${target}eabihf"
   fi
 
-  echo "${target}"
+  printf '%s' "${target}"
```

Comment:
As for penalising shells with no built-in `printf`, there’s a nice way you can work around this for Korn shells at least:

```sh
if test -n "${KSH_VERSION:-}"; then
    write() {
        print -nr -- "$*"
    }
    writeln() {
        print -r -- "$*"
    }
else
    write() {
        printf '%s' "$*"
    }
    writeln() {
        printf '%s\n' "$*"
    }
fi
```

Then use the `write` and `writeln` helper functions everywhere a simple `echo` (see my previous comment) won’t do. This will be fast on all Korn shell variants (that set `$KSH_VERSION`; some set it as alias to `${.sh.version}` but you cannot check *that* portably) and on all shells with `printf` built in, which covers most (including many ash variants such as busybox and dash).